perfmetrics/scripts/testing_on_gke/examples/run-automated.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used purely for automating the run of
+# the script run-gke-tests.sh to
+# run it periodically as a cron-job.
+#
+# For your case, add/remove/modify the configuration parameters as you need.
+#
+# Assumptions for this script to work:
+# 1. You have appropriate access to project_id defined below.
+# 2. You have the cluster with cluster_name defined below, or enough.
+# resources in project_id to create this cluster.
+
+# Print all shell commands.
+set -x
+
+# Fail if any command fails.
+set -e
+
+# Define configuration parameters.
+export project_id=gcs-fuse-test-ml
```

Comment:
This should come from the job. They should not be hardcoded here. I would add this as PII.

Reply:
done.. removed all PII.